Tests/Server/JokesServiceTests.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using FluentAssertions;
+using Moq;
+using JokesOnYou.Web.Api.Repositories.Interfaces;
+using JokesOnYou.Web.Api.Services;
+using JokesOnYou.Web.Api.Models;
+using JokesOnYou.Web.Api.DTOs;
+using AutoMapper;
+using JokesOnYou.Web.Api.Profiles;
+
+namespace Tests.Server
+{
+    public class JokesServiceTests
+    {
+        private readonly IMapper _mapper = new Mapper(new MapperConfiguration(cfg => cfg.AddProfile<JokesProfile>()));
+
+        [Fact]
+        public void GetAllUsers_Returns_ListOfJokesDto()
+        {
+            var repo = new Mock<IJokesRepository>();
+            var service = new JokesService(repo.Object, _mapper);
```

Comment:
Use the constructor as a setup method, it will drastically reduce the amount of code in our tests